tools/cockpit.spec

```diff
@@ -60,6 +65,7 @@ URL:            https://cockpit-project.org/
 Version:        0
 Release:        1%{?dist}
 Source0:        https://github.com/cockpit-project/cockpit/releases/download/%{version}/cockpit-%{version}.tar.xz
+Source1:        https://github.com/cockpit-project/cockpit/releases/download/%{version}/cockpit-node-%{version}.tar.xz
```

Comment:
In other projects, this is conditional, no?  Update: Ahh, I see, it would break packit.